auth/v1.js

```diff
@@ -637,7 +739,9 @@ api.declare({
         lastModified: new Date().toJSON(),
         lastDateUsed: new Date().toJSON(),
         lastRotated:  new Date().toJSON()
-      }
+      },
+      scopes:         [],
+      disabled:       0
```

Comment:
I think it's safer to just delete this API end-point... we're not going to be using it anyways..

It was for legacy import, and we don't have export anymore...